packages/@ama-sdk/core/src/fwk/logger.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Logger Client interface.
+ *
+ * Duplicate from [@o3r/core]{@link https://github.com/AmadeusITGroup/otter/blob/main/packages/%40o3r/core/src/log/logger.ts}
+ * All modifications should be made in both interfaces
+ */
+export interface Logger {
+  /**
+   * Log an error.
+   * @param message Message to log
+   * @param optionalParams Optional parameters to log
+   */
+  error(message?: any, ...optionalParams: any[]): void;
```

Comment:
Maybe it could be valuable to have the `this: Logger` as first param to avoid le context loosing usage